.github/workflows/release.yml

```diff
@@ -24,42 +24,15 @@ jobs:
           go-version: 1.19.5
 
       - name: Run GoReleaser
-        # GoReleaser v4.2.0
-        uses: goreleaser/goreleaser-action@f82d6c1c344bcacabba2c841718984797f664a6b
+        # GoReleaser v2.5.0
```

Comment:
Is the downgrade intentional? I remember Praveen mentioning that there is another PR up for this, if that is the case this looks good!

Reply:
Yes, We have seen issues with alpha releases with `4.2.0`.